controller/todo-controller.js

```diff
@@ -1,39 +1,41 @@
 
 const todoService = require("../service/todo-service")
 
-
 // Create Todo
 const createTodo = async (req, res) => {
-    try {
-        const {message} = req.body;
-        const userId = req.params.userId;
-        const date = new Date();
-
-        const todo ={
-            userId : userId,
-            todoId : Math.floor(Math.random() * 100) + userId,
-            message : message,
-            createdAt : date
+   try {
+        const {title, message} = req.body;
+        const {userId} = req.params;
```

Comment:
get userId from token instead params

Reply:
In middelware userId is extracted from token and saved as req.params